generic/tests/netperf.py

```diff
@@ -534,27 +521,17 @@ def get_state():
             state_list.append('intr')
             state_list.append(ninit)
 
-        if arch.ARCH in ('ppc64', 'ppc64le'):
-            exits = int(ssh_cmd(host, "cat /sys/kernel/debug/kvm/exits"))
-            state_list.append('exits')
-            state_list.append(exits)
-        else:
-            io_exits = int(ssh_cmd(host, "cat /sys/kernel/debug/kvm/io_exits"))
-            irq_injs = int(
-                ssh_cmd(host, "cat /sys/kernel/debug/kvm/irq_injections"))
-            state_list.append('io_exits')
-            state_list.append(io_exits)
-            state_list.append('irq_injs')
-            state_list.append(irq_injs)
+        exits = int(ssh_cmd(host, "cat /sys/kernel/debug/kvm/exits"))
+        state_list.append('exits')
+        state_list.append(exits)
 
         return state_list
 
     def netperf_thread(i, numa_enable, client_s, timeout):
         cmd = ""
         fname = "/tmp/netperf.%s.nf" % pid
         if numa_enable:
-            output = ssh_cmd(client_s, "numactl --hardware")
-            n = re.findall(r"node (\d+) cpus:", output)[-1]
+            n = abs(int(params.get("numa_node"))) - 1
```

Comment:
If you want to use the numa node that you specified, why minus by 1?

Reply:
For keep consistent with virttest/qemu_vm.py

        # Add numa memory cmd to pin guest memory to numa node
        if params.get("numa_node"):
            numa_node = int(params.get("numa_node"))
            if len(utils_misc.get_node_cpus()) < int(params.get("smp", 1)):
                logging.info("Skip pinning, no enough nodes")
            elif numa_node < 0:
                n = utils_misc.NumaNode(numa_node)
                cmd += "numactl -m %s " % n.node_id
            else:
                n = numa_node - 1
                cmd += "numactl -m %s " % n

Reply:
@luckyh , any comments ?

Reply:
No more, thanks for your explain.